src/Route/CmsPageRouter.php

```diff
@@ -19,7 +19,9 @@
 use Sonata\PageBundle\Request\SiteRequestContextInterface;
 use Sonata\PageBundle\Site\SiteSelectorInterface;
 use Symfony\Cmf\Component\Routing\ChainedRouterInterface;
+use Symfony\Cmf\Component\Routing\RouteObjectInterface;
```

Comment:
is it possible to class/interfaces from `Symfony\Component\Routing` ?
we should keep using one routing component and IMO we should keep using things from symfony routing only!

Reply:
I don't understand your point, `RouteObjectInterface` is only defined in `Symfony\Cmf`, you prefer to use string (cmf_routing_object)  value instead on class constant ?

Reply:
I don't understand either.

Reply:
No, I mean, we don't depends of Symfony cmf anymore!

Reply:
I mean

> is it possible, we don't use class/interfaces from Symfony\Component\Routing ?

sorry 😄